FF108 RelNote: Import maps are supported (#22888)
mdn/content: files/en-us/mozilla/firefox/releases/108/index.md

### APIs

> - [Import maps](/en-US/docs/Web/HTML/Element/script/type/importmap) are now supported.

Reviewer: Should we add some context how it can be used?

```suggestion
- [Import maps](/en-US/docs/Web/HTML/Element/script/type/importmap) are now supported, which indicate that the body of a [<script> element](/en-US/docs/Web/HTML/Element/script) contains an import map.
```

Reviewer: Sure thing

>   These allow more flexible and ergonomic module specifiers when importing [JavaScript modules](/en-US/docs/Web/JavaScript/Guide/Modules).

Reviewer: Minor reword suggestion:

```suggestion
  Import maps provide flexibility and additional control over how the browser resolves module specifiers when importing [JavaScript modules](/en-US/docs/Web/JavaScript/Guide/Modules).
```

Author: ```suggestion
  Import maps provide flexibility and additional control over how browsers resolve module specifiers when importing [JavaScript modules](/en-US/docs/Web/JavaScript/Guide/Modules).
```
